lib/usdUfe/utils/Utils.cpp

```diff
@@ -42,13 +42,16 @@ std::string prettifyName(const std::string& name)
             if ((i > 0 && (i < (nbChars - 1))
                  && (!std::isupper(name[i + 1]) && !std::isdigit(name[i + 1])))
                 || std::islower(name[i - 1])) {
-                prettyName += ' ';
+                if (prettyName.size() > 0 && prettyName.back() != ' ') {
```

Comment:
Fixes two observed cases where a double space was added because an explicit space character "_" was followed by a lower to upper case transition.